packages/gatsby/src/db/nodes.js

```diff
@@ -51,3 +49,9 @@ module.exports.loadNodeContent = node => {
     })
   }
 }
+
+module.exports = {
+  ...nodesDb,
+  runQuery: runFastFiltersAndSort,
```

Comment:
I might propagate this change through (`runQuery` -> `runFastFiltersAndSort`) but that's gonna result in a much bigger PR so I stopped here.